faceswap.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python3
+import argparse
+from scripts.extract import ExtractTrainingData
+from scripts.train import TrainingProcessor
+from scripts.convert import ConvertImage
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    subparser = parser.add_subparsers()
+    extract = ExtractTrainingData(subparser, "extract", "Extract the faces from a pictures.")
+    convert = ConvertImage(subparser, "convert", "Convert a source image to a new one with the face swapped.")
```

Comment:
Probably it will be more clear to save the order `extract` -> `train` -> `convert` here to avoid confusion (because as far as I understand this order also affects the order in auto-generated help output) :)